public/js/applicantModal.js

```diff
@@ -170,7 +170,9 @@ export function addEventListenersToDisplayApplicantModal() {
                             displayField(data, 'additionalNotes', 'No notes')
                             data.chosenCourseDate = prettyDate(data.chosenCourseDate)
                             displayField(data, 'chosenCourseDatePretty', 'Not asked yet')
-                            displayField(data,'userProfileLink', 'No Link Yet' )
+                            if (!studentUrl) {
+                                displayField(data, 'userProfileLink', 'No Link Yet')
+                            }
```

Comment:
given this is unnecessary as it stands, id rather remove this so there isnt code in here that does nothing, as it may confuse other devs

Reply:
I gort rid of it